Created taleno-digital.md (#4651)
PostHog/posthog.com: contents/partners/taleno-digital.md

---
title: taleno.digital
showTitle: true
hideLastUpdated: true
---

**[← Back to partners](/partners)**

> [taleno.digital](https://taleno.digital) is a Data Consultancy service that offers cost effective yet high quality Modern Data Stack solutions for Data Warehousing and Product Analytics.

Reviewer: ```suggestion
[taleno.digital](https://taleno.digital) is a data consultancy service that offers cost effective, high quality solutions for data warehousing and product analytics built on the modern data stack.
```

> Helping you get up to speed in setting up using PostHog Cloud only and getting insights so you can focus on data-driven product iteration.

Reviewer: ```suggestion
We can help you get set up with PostHog Cloud and start creating insights which enable you to focus on data-driven product iteration.
```

## Services offered

### App

> taleno.digital is your analytics engineer-as-a-service so you don't need to hire an FTE. 

Reviewer: ```suggestion
taleno.digital is your analytics engineer-as-a-service, so you don't need to hire an FTE. 
```

### Support

- Setup PostHog Cloud & Deployment
- Handle issues / inconsistencies encountered with PostHog Cloud

> - Creation of well thought Actions, Insights & Dashboards

Reviewer: ```suggestion
- Creation of Actions, Insights & Dashboards
```

### Pricing

> - The client should cover the monthly subscription of their PostHog Cloud on top of 1M FREE events per Month.

Reviewer: ```suggestion
- The client should cover the cost of their PostHog Cloud subscription on top of 1M free events per month
```

> - Get 30% discount if your product is B2C.

Reviewer: ```suggestion
- Get a 30% discount if your product is B2C
```

> - You only need to pay for the initial setup cost of $100 and $100 per succeeding request.

Reviewer: ```suggestion
- You only need to pay for the initial setup cost of $100, and $100 per subsequent request
```

## Contact

> posthog@taleno.digital

Reviewer: ```suggestion
Get in touch by emailing [posthog@taleno.digital](mailto:posthog@taleno.digital)
```
